packages/services/src/balances/sip10-balances.service.ts

```diff
@@ -38,7 +38,7 @@ export class Sip10BalancesService {
     @inject(Types.SettingsService) private readonly settingsService: SettingsService,
     private readonly stacksApiClient: HiroStacksApiClient,
     private readonly marketDataService: MarketDataService,
-    private readonly sip10TokensService: Sip10AssetService
+    private readonly sip10AssetsService: Sip10AssetService
```

Comment:
@alexp3y initially I thought I would need to extend this service to give me `asset` based on `tokenId` / `assetId`. 

I didn't need to do that but I kept this renaming, I think its a bit clearer if we just go with `asset` and banish `token`? 🤔